database/scripts/seed.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+const Soundtrack = require('../models/soundtracks.js');
+
+mongoose.connect('mongodb://localhost/bonuses', {
+  autoIndex: false,
+  dbName: 'bonuses',
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}).then( () => {
+  console.log('Connection to MongoDB successful!');
+}).catch( (err) => {
+  console.log('DB connection error:', err);
+});
+
+let albums  = [];
+
+const fakeSongList = () => {
+  let songs = [];
+  let listLength = Math.floor(Math.random() * 13) + 9;
+
+  for (let j = 0; j < listLength; j++) {
+    let number = j.toString();
+
+    if (j < 10) {
+      number = number.padStart(2, "0");
+    }
+
+    let song = {
+      track_number: number,
+      name: faker.lorem.words(),
+      song: faker.internet.url()
+    }
+    songs.push(song);
+  }
+  return songs;
+}
+
+for (let i = 1; i <= 10; i++) {
```

Comment:
This `10` seems to be a typo? The `i` here directly seeds the `bundle_id` key in your database (line 43 below), so I would expect this loop to reach 100. 

If, on the other hand, you plan to only have 10 total bonuses and then mix-and-match them from bundle to bundle, I'd rename the `bundle_id` key to something else.

Reply:
I forgot to update from my test number to actual bundle number (100).